src/Search.js

```diff
@@ -1,11 +1,10 @@
 import React, { useState } from "react";
 import SearchResults from "./SearchResults";
-const SearchButton = () => {
+const SearchButton = props => {
```

Comment:
Does this component actually represent a `SearchButton`? What do you think it should be renamed to?

Reply:
Search is better. 
 ;)